integration-tests/_tests/fixtures/hello-world-fixture.test.ftd

```diff
@@ -0,0 +1,13 @@
+-- import: fastn
+
+-- fastn.test: Hello world fixture
+
+-- fastn.get: Fetching Test Data (from test server)
+url: /get-data/
+http-status: 200
+
+-- fastn.get.test:
+
+fastn.assert.eq(fastn.http_response["data"], "Hello, World!");
+
+-- end: fastn.test
```

Comment:
Do we need this end statement? We can change the type of `fastn.test` to just a component without children.

Reply:
Removed children from fastn.test component